tests/system_tests_one_router.py

```diff
@@ -279,6 +279,68 @@ def setUpClass(cls):
         ])
         cls.routers.append(cls.tester.qdrouterd(name, config_9, wait=False, expect=Process.EXIT_FAIL))
 
+        # A tcpConnector must supply an 'address' value
```

Comment:
Should we add these same tests but have these configs added via qdmanage and see if those configs get rejected by the router (check if the router returns a `BadRequestStatus`) ?

Reply:
Great minds think alike - I've got some skmanage based tests that do exactly that.  These will be part of my next patch, where the separation of TLS domain and session is done.

It's part of that patch because the error descriptions are changed in that patch for clarification and the new tests check for those error messages.

Reply:
> Great minds think alike 

Um

Reply:
> 
> It's part of that patch because the error descriptions are changed in that patch for clarification and the new tests check for those error messages.

Sounds good.